tests/rados/test_mon_addition_removal.py

```diff
@@ -59,6 +59,15 @@ def run(ceph_cluster, **kw):
             )
             raise Exception("Mon in quorum error post removal")
 
+        # Checking for crashes in the cluster
+        crash = rados_obj.do_crash_ls()
```

Comment:
Please change this to `ceph crash ls-new` so that any residual crashes reported previously are not considered during this execution

Reply:
Updated the command.